### PR TITLE
Dual 5015 blower fan part cooling duct/mount for V0

### PR DESCRIPTION
**Pull request by designbydave**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
./designbydave/5015_Fan_Ducts_for_V0

#### How was it tested? 
I printed them, installed them and printed several parts with this confirgration

#### Any background context you want to provide?

V0 stock 3010 part cooling fans not enough for printing/cooling PLA, so we add more blow-ie

#### Screenshots (if appropriate)
pics in directory

#### Further notes
first time using github, so, sorry if I'm doing something incorrectly. I probably am. It was suggested by a couple members of the discord server to upload this mod here.

### CODE REVIEW

**Comment by blalor** [comment]
This looks interesting. How's it fare with fast accelerations? Seems like the un-braced mass of the fans could cause them to break the ducts. Would tying them together at the top make sense?

**Comment by designbydave** [comment]
> 
> 
> This looks interesting. How's it fare with fast accelerations? Seems like the un-braced mass of the fans could cause them to break the ducts. Would tying them together at the top make sense?

I was never able to test it with anything over ~5000 mm/s^2 as the steppers on my V0 would loose steps. Maybe the extra mass of the fans and ducts made that situation worse. So, it seems to drive really high accelerations you will need the more powerful steppers of the V0.1.

I don't think the ducts will break, but there could be some unwanted flexing or motion of the fans. I did consider a spacer to tie the fans together. A simple printed cylinder between the unused mounting holes would be easy to make and install.

File: printer_mods/README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ like so:
 |Dan88 |[V0 one-part Skirts](./Dan88/V0skirt) |One part Skirts for the V0 |:heavy_check_mark: |:x: |:x: |:x:|
 |DC67|[Plug Panel Mod for Schaffner FN286-10-06](./DC67/plug_panel_schaffner)|A modified plug panel for the Power Entry Connector Schaffner FN286-10-06 with two integreated Keystone sockets. |:x:|:grey_question:|:heavy_check_mark:| :x: |
 |deepfriedheroin |[V2 Bed Wagos](./deepfriedheroin/v2_bed_wagos) |Wire management for the v2 bed |:x: |:x:|:heavy_check_mark: |:x:|
+| designbydave | [5015 Part Cooling Fan Duct/Mounts for V0](./designbydave/5015_Fan_Ducts_for_V0) | 5015 Part Cooling Fan Duct/Mounts for V0 | :heavy_check_mark: | :x: | :x: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:85]: Can you add an `:x:` for the Trident column?

File: printer_mods/designbydave/5015_Fan_Ducts_for_V0/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Overview
+
+Left and Right side ducts/mounts for 5015 blower part cooling fans for Voron V0. 
+
+# BOM
+
+2x M3x20mm BHCS
```
Comment (KiloQubit) [inline printer_mods/designbydave/5015_Fan_Ducts_for_V0/README.md:7]: Can you make this a list, so it doesn't all display on a single line?

```
- 2x M3x20mm BHCS
- 2x M3x50mm BHCS or SHCS
- 4x M3 nuts
```

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Looks like you have some unprintable bridging around the bottom of the fan mount hole
Screen Shot 2021-09-19 at 10 33 29

Is this supposed to be a removable support?  If so, where do you cut it on the right side?  The left side has an obvious gap, but the right side does not.  Also add instructions to the readme to remove it.
Screen Shot 2021-09-19 at 10 35 57

**Comment by bdbell** [comment]
I don't see any updates or commits after the request for changes.  Will those changes be coming?

**Comment by FHeilmann** [comment]
Comments of reviewers were not adressed. Closing as stale, most importantly because at this point this mod is better off being resubmitted than reviving this PR.
If there are any questions, feel free to reach out.

**Pull request closed without merging**